File: src/main/java/catwalks/item/ItemCatwalkTool.java
package catwalks.item;

import net.minecraft.entity.player.EntityPlayer;
import net.minecraft.item.ItemStack;
import net.minecraft.util.math.BlockPos;
import net.minecraft.world.IBlockAccess;

public class ItemCatwalkTool extends ItemBase {

	public ItemCatwalkTool() {
		super("tool");
		setUnlocalizedName("catwalktool");
	}
	
	@Override
	public boolean doesSneakBypassUse(ItemStack stack, IBlockAccess world, BlockPos pos, EntityPlayer player) {
		return true;
	}
	
}

File: src/main/java/catwalks/item/ItemCatwalkTool.kt
package catwalks.item

import net.minecraft.entity.player.EntityPlayer
import net.minecraft.item.ItemStack
import net.minecraft.util.math.BlockPos
import net.minecraft.world.IBlockAccess

class ItemCatwalkTool : ItemBase("tool") {

    init {
        unlocalizedName = "catwalktool"
        setMaxStackSize(1)
    }

    override fun doesSneakBypassUse(stack: ItemStack?, world: IBlockAccess?, pos: BlockPos?, player: EntityPlayer?): Boolean {
        return true
    }

}
